src/core.js

```diff
@@ -1481,9 +1481,11 @@ define([
             }
             if (selected && selected.length > 0) {
                 this.jstree('select_node', selected[0]);
+                selected[0].scrollIntoView();
```

Comment:
Did you consider using our scroll function: `this.scrollTreeTo(ufid)`

It looks like we're not using `scrollIntoView` anywhere else.

Reply:
Updated: https://github.com/dimagi/Vellum/pull/954/commits/e0f0dc35a8e8a399ff3ab03aa201f207da46a981